tests/tools/commands/test_repository.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import
+
+import os
+import signal
+import subprocess
+from textwrap import dedent
+
+import pytest
+
+from pex.common import safe_open, temporary_dir
+from pex.testing import run_pex_command
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any, Dict, Iterator
+
+
+FABRIC_VERSION = "2.5.0"
+
+
```

Comment:
Could you please add a test for the `info` subcommand?

Reply:
Yup, will do.

Reply:
Done.